src/index.css

```diff
@@ -15,3 +15,34 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+:root {
+  --black-gradient: linear-gradient(
+    144.39deg,
+    #ffffff -278.56%,
+    #adadad -78.47%,
+    #d1d1d1 91.61%
+  );
+  --card-shadow: 0px 20px 100px -10px rgba(215, 215, 215, 0.1);
+}
+
+* {
+  scroll-behavior: smooth;
+}
+
+.feature-card:hover {
+  background: var(--black-gradient);
+  box-shadow: var(--card-shadow);
+}
+
+.feedback-container .feedback-card:last-child {
+  margin-right: 0px;
+}
+
+.feedback-card {
+  background: transparent;
+}
+
+.feedback-card:hover {
+  background: var(--black-gradient);
+}
```

Comment:
We wont really use css code, if we will absolutely have to, we will use scss, so please try to do whatever you did on here with tailwind. If you cant, please open another .scss file in the components folder and type your scss code there.